stock_move_entries/model/stock_move_entries.py

```diff
@@ -30,11 +30,19 @@
 class AccountMoveLine(osv.Model):
     _inherit = "account.move.line"
 
-    """
-    """
+    def _auto_init(self, cr, context=None):
+        res = super(AccountMoveLine, self)._auto_init(cr, context=context)
+        index = 'account_move_line_sm_id_account_id_index'
+        cr.execute("SELECT indexname FROM pg_indexes WHERE indexname = %s",
+                   (index,))
+        if not cr.fetchone():
+            cr.execute(
+                'CREATE INDEX %s ON account_move_line (account_id, sm_id)' %
```

Comment:
That is cool!
But we are not sure if this module is used with postgresql >=9.5 for all cases.

IMHO we should use the full compatibility way.